README.md

```diff
@@ -74,7 +74,7 @@ You can directly copy and paste all the commands from the text given here into t
         c_pass VARCHAR(10) NOT NULL,
         Name VARCHAR(20) NOT NULL,
         Address VARCHAR(20) NOT NULL,
-        Pincode NUMBER(6) NOT NULL,
+        Pincode int NOT NULL,
```

Comment:
A pincode is always of 6 digits, so NUMBER(6) would be a better choice. Open for your opinion.